eng/Directory.Build.Data.props

```diff
@@ -84,7 +84,7 @@
     <DelaySign>false</DelaySign>
     <PublicSign>false</PublicSign>
     <ImportDefaultReferences>false</ImportDefaultReferences>
-    <UseProjectReferenceToAzureCore>true</UseProjectReferenceToAzureCore>
+    <UseProjectReferenceToAzureCore>false</UseProjectReferenceToAzureCore>
```

Comment:
Nice catch!

Seems like this should be the default state at the `eng` level, with individual libraries overriding if they need the latest and greatest.

Reply:
My understanding is that we wanted it to be `true` while we were all adopting the changes that were coming to Core. 
Now that we are getting ready to ship, each client library will be able to override, like you say

Reply:
There was some discussion in the past about setting this to true in the Azure.Core.props file when you are building in VS.  That has potential issues if things get out of sync but while things are in sync we can use a package reference for our command line/official builds and use a project reference for local devs that are working in VS.